wgsl/index.bs

```diff
@@ -1779,6 +1795,14 @@ The valid [=enable-extensions=] are listed in the following table.
       <td>[[WebGPU#dom-gpufeaturename-dual-source-blending|"dual-source-blending"]]
       <td>The attribute [=attribute/blend_src=] is valid to use in the WGSL module. Otherwise, using
           [=attribute/blend_src=] will result in a [=shader-creation error=].
+  <tr><td><dfn noexport dfn-for="extension">`subgroups`</dfn>
+      <td>[[WebGPU#gpufeaturename|"subgroups"]]
+      <td>It is valid to use [[#builtin-inputs-outputs|subgroup built-in values]],
+          [[#subgroup-builtin-functions|subgroup built-in functions]], and
+          [[#quad-builtin-functions|quad built-in functions]] in a WGSL module.
+          Otherwise, any use will result in a [=shader-creation error=].
+
+          Issue: replace feature link when extension lands.
```

Comment:
Now is the time to fix this, right?

Reply:
Unfortunately the specs need to land in main to do this linking correctly.

Reply:
You can add it to the `<pre class=anchors>` block to force it to link even though it doesn't exist in the published spec yet. I prefer that since it makes the problem invisible to readers.